Source/src/components/ComponentAnimation.cpp

```diff
@@ -204,6 +226,13 @@ void Hachiko::ComponentAnimation::DrawGui()
 
                 strcpy_s(name, state_machine->GetClipName(clip_idx).c_str());
 
+                // Don't show clip "none"
+                if (strcmp(name, "none") == 0)
```

Comment:
Same with this `"none"` let's make it use the same variable I've described above.